piwigo-nginx-site

```diff
@@ -62,19 +62,49 @@ server {
 		deny all;
 	}
 
-	location /upload {
+	location ^~ /upload {
 		internal;
 	}
 
-	location /galleries {
+	location ^~ /galleries {
 		internal;
 	}
 
-	location /_data {
+	location ^~ /_data {
 		internal;
 	}
 
 	location /_data/combined {
 		try_files $uri $uri/ =404;
 	}
-}
\ No newline at end of file
+	
+	# Various additional fixes if the extra security blocks above are used.
+	
+	# Enable the "Display reference file" link of the LocalFiles Editor plugin
+	location = /plugins/LocalFilesEditor/show_default.php {
+		try_files  $uri =404;
+		fastcgi_pass unix:/run/php-fpm/www.sock;
+		fastcgi_index  index.php;
+		fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
+		fastcgi_param  SCRIPT_NAME  $fastcgi_script_name;
+		include fastcgi_params;
+	}
+		
+	# Enable exif metadata testing using /tools/metadata.php as described in
```

Comment:
I think this should probably come after the `location = /plugins/piwigo_privacy/get.php` block
Also better to give generic explanation that if you use the security block this is how you can open other php routes but leave the actual block commented out.
This is also apply for the metadata block. I don't think we need a block for each file
But just give the tools folder as another location with files that are worth considering